### PR TITLE
Fix RNVP, MAF and MADE diagrams not showing white version in dark mode

### PR DESCRIPTION
**Pull request by janosh**
[From GH docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to):

> The old method of specifying images based on the theme, by using a fragment appended to the URL (#gh-dark-mode-only or #gh-light-mode-only), is deprecated and will be removed in favor of the new method described above.

### CODE REVIEW

**Comment by janosh** [comment]
Still seeing the light mode version of diagrams in dark-themed GitHub due to https://github.com/github/markup/issues/1583.

| Expected                                                                                                                                    | Actual                                                                                                                                      |
| ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
| Screenshot 2022-11-29 at 10 45 00 | Screenshot 2022-11-29 at 10 42 54 |

Tl;dr: GitHub just uses a media query that matches the system color mode which is independent of the actual color mode set in GitHub personal settings. Could be set to "Dark Dimmed" (like I have) but if OS is in light mode, that's what the media query will return.

Screenshot 2022-11-29 at 10 43 57
